carlos-calc/src/Keypad.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+class Keypad extends React.Component {
+    render() {
+        const {calculate, handleClick, addition, subtract, division, multiplication, negative, reset} = this.props;
+        return (
+            <section className="keypad">
+                <button className="numbers"onClick= {handleClick} value = "1" >1</button>
+                <button className="numbers"onClick= {handleClick} value = "2">2</button>
+                <button className="numbers"onClick= {handleClick} value = "3">3</button>
+                <button className = "operators" onClick= {addition} value = "+">+</button><br/>
+                <button className="numbers"onClick= {handleClick} value = "4">4</button>
+                <button className="numbers"onClick= {handleClick} value = "5">5</button>
+                <button className="numbers"onClick= {handleClick} value = "6">6</button>
```

Comment:
An alternative would be to handle the values in the onClick. 
onClick={() => handleClick("6") }